src/mozanalysis/udf.py

```diff
@@ -0,0 +1,121 @@
+from pyspark.sql import functions as F
+from pyspark.sql.functions import udf
+from pyspark.sql.types import DoubleType, LongType, StructField
+
+from statsmodels.stats.weightstats import DescrStatsW
+
+
+@udf(LongType())
+def histogram_sum(values):
+    """Returns the weighted sum of the histogram.
+
+    Note that this is the sum *post*-quantization, which amounts to a
+    left-hand-rule discrete integral of the histogram. It is therefore
+    likely to be an underestimate of the true sum.
```

Comment:
It would be interesting to explore what we'd need to have in order to either a) make this a trapezoidal-rule integration or b) aggregate the high-resolution histogram sums that the clients report.

For (a), we'd have to know the name of the column and generate the list of bins. This is maybe out of scope right now.